zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -255,3 +269,48 @@ def get_parse_product_request(
         )
         scrapy_request.meta["allow_offsite"] = True
         return scrapy_request
+
+
+class ExperimentalEcommerceSpiderParams(BaseModel):
+    urls: List[str] = Field(
+        title="URLs",
+        description=(
+            "Initial URLs for the crawl, separated by new lines. Enter the "
```

Comment:
I'm okay in having a new input for multi-URL newline-delimited values which completely replaces the old one. 

We've already agreed with the Scrapy Cloud team that this would be presented in a new textbox widget where multi-line values looks okay.